cmpirque/static_pages/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
The views should be a folder too

Reply:
I will change it to cmpirque/pages/views